tests/by-util/test_od.rs

```diff
@@ -579,6 +579,18 @@ fn test_invalid_offset() {
     new_ucmd!().arg("-Ab").fails();
 }
 
+#[test]
+fn test_offset_compatibility() {
+    let input = [0u8; 4];
+
+    new_ucmd!()
+        .arg("-Anone")
+        .run_piped_stdin(input)
+        .success()
+        .stderr_is("od: warning: Ignored superfluous character(s) after radix specifier 'n'\n")
```

Comment:
Why do you expect a warning? GNU `od` doesn't seem to show a warning:
```
$ cat input.bin | od -Anone
 000000 000000
```

Reply:
Thanks, I did not know that level of compatibility was the goal. I've removed the warning.